admin/lib/accounts/add.js

```diff
@@ -1,24 +1,21 @@
 module.exports = add
 
-var request = require('../../../helpers/request')
-var deserialise = require('../../../helpers/deserialise')
-var serialise = require('../../../helpers/serialise')
+var request = require('../../../utils/request')
+var deserialise = require('../../../utils/deserialise')
+var serialise = require('../../../utils/serialise')
 
 function add (state, account, options) {
   return request({
     url: state.url + '/accounts' + query(options),
     method: 'POST',
     headers: {
-      'Accept': 'application/vnd.api+json',
-      'Authorization': 'Bearer ' + state.session.id,
-      'Content-Type': 'application/vnd.api+json'
+      authorization: 'Bearer ' + state.session.id
     },
-    body: JSON.stringify(serialise('account', account))
```

Comment:
moved `accept` and `content-type` header to our request helper method.

I also added `{json: true}` to the request options in the helper, so we don’t need to JSON stringify or parse